woocommerce/woocommerce-android
# Experiments/remove ab test jetpack timeout override

### Description
Removes code related to Jetpack timeout policy override. The timeout value used for site verification will be always 15s from now on. 

The experiment has been marked as complete in Firebase Console

### Testing instructions
Run the app and check that site verification from Site picker screen runs as expected.
